sklearn/datasets/_lfw.py

```diff
@@ -116,8 +116,16 @@ def _check_fetch_lfw(data_home=None, funneled=True, download_if_missing=True):
 
 def _load_imgs(file_paths, slice_, color, resize):
     """Internally used to load images"""
-    # import PIL only when needed
-    from ..externals._pilutil import imread, imresize
+    try:
+
```

Comment:
```suggestion
```
Is there a reason for having this blank line that I don't know of?

Reply:
There is no reason. I guess `black` does not have an opinion on this newline. 😅